crytic_compile/crytic_compile.py

```diff
@@ -68,8 +68,9 @@ def _extract_libraries(libraries_str: Optional[str]) -> Optional[Dict[str, int]]
     matches = re.findall(pattern, libraries_str)
 
     if not matches:
-        logging.info(f"Libraries {libraries_str} could not be parsed")
```

Comment:
This is using `logging` instead of `LOGGER` so it didn't raise any warning. I think it's more appropriate to reject the input with a `ValueError` otherwise commands like `print-libraries` will keep running and be difficult to debug.